### ISSUE TITLE
Accordion on guild/party page is inaccessible to keyboard navigation

### ISSUE THREAD

**Issue by J-Sanderson** [HabitRPG/habitica#12653]
Opening as advised in discussion for issue #11067

The accordion menu at the right hand side of the guild and party page uses clickable divs instead of buttons and cannot be accessed via keyboard navigation.

**Comment by J-Sanderson** [comment]
I'm willing to pick this one up too.

**Comment by paglias** [comment]
@J-Sanderson thanks! I've marked it as in progress
